editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -744,13 +744,16 @@ export const NavigatorItem: React.FunctionComponent<
 
   const currentlyRenaming = EP.pathsEqual(props.renamingTarget, props.navigatorEntry.elementPath)
 
-  const onClick = React.useCallback(
+  const onMouseDown = React.useCallback(
     (e: React.MouseEvent) => {
-      if (isRenderPropNavigatorEntry(navigatorEntry)) {
-        e.stopPropagation()
-      }
+      // Avoid doing any of this for a right click.
+      if (e.button !== 2) {
```

Comment:
Should this actually be `e.button === 0`?

Reply:
Yeah, I think I was just thinking in terms of when it shouldn't trigger because of the context menu.

Reply:
Fixed in 98f637f45.